config.yaml

```diff
@@ -1,1055 +1,5 @@
-  ## made -- and any additional spans that arrive for traces that were
-  ## active during Stress Relief will respect the decisions made during
-  ## that time.
-  ## The measurement of stress is a lagging indicator and is highly
-  ## dependent on Refinery configuration and scaling. Other configuration
-  ## values should be well tuned first, before adjusting the Stress Relief
-  ## Activation parameters.
-  ## Stress Relief is not a substitute for proper configuration and
-  ## scaling, but it can be used as a safety valve to prevent Refinery from
-  ## becoming unstable under heavy load.
-  ##
-  ####
-    ## Mode is a string indicating how to use Stress Relief.
-    ##
-    ## Sets the stress relief mode. "never" means that Stress Relief will
-    ## never activate "monitor" is the recommended setting, and means that
-    ## Stress Relief will monitor the status of refinery and activate
-    ## according to the levels set below. "always" means that Stress Relief
-    ## is always on, which may be useful in an emergency situation.
-    ##
-    ## default: never
-    ## Eligible for live reload.
-    # Mode: never
-
-    ## ActivationLevel is the stress_level (from 0-100) at which Stress
-    ## Relief is triggered.
-    ##
-    ## Sets the stress_level (from 0-100) at which Stress Relief is
-    ## triggered.
-    ##
-    ## default: 90
-    ## Eligible for live reload.
-    # ActivationLevel: 90
-
-    ## DeactivationLevel is the stress_level (from 0-100) at which Stress
-    ## Relief is turned off.
-    ##
-    ## Sets the stress_level (from 0-100) at which Stress Relief is turned
-    ## off (subject to MinimumActivationDuration). It must be less than
-    ## ActivationLevel.
-    ##
-    ## default: 70
-    ## Eligible for live reload.
-    # DeactivationLevel: 70
-
-    ## SamplingRate is the sampling rate to use when Stress Relief is
-    ## activated.
-    ##
-    ## Controls the sampling rate to use when Stress Relief is activated. All
-    ## new traces will be deterministically sampled at this rate based only
-    ## on the traceID. It should be chosen to be a rate that sends fewer
-    ## samples than the average sampling rate Refinery is configured to use.
-    ## For example, if Refinery is configured to normally sample at a rate of
-    ## 1 in 10, then Stress Relief should be configured to sample at a rate
-    ## of at least 1 in 30.
-    ##
-    ## default: 100
-    ## Eligible for live reload.
-    # SamplingRate: 100
-
-    ## MinimumActivationDuration is the minimum time that stress relief will
-    ## stay enabled.
-    ##
-    ## Setss the minimum time that stress relief will stay enabled, once
-    ## activated. This helps to prevent oscillations.
-    ##
-    ## Accepts a duration string with units, like "10s".
-    ## default: 10s
-    ## Eligible for live reload.
-    # MinimumActivationDuration: "10s"
+  ConfigurationVersion: 2
+  MinRefineryVersion: v2.0
```

Comment:
Are these deletions expected?

Reply:
Yes; some changes crept in that shouldn't have, but this is the correct regeneration.